ksqldb-engine/src/main/java/io/confluent/ksql/execution/json/PlanJsonMapper.java

```diff
@@ -32,19 +32,23 @@
 
   INSTANCE;
 
-  private final ObjectMapper mapper = new ObjectMapper()
-      .registerModules(
-          new Jdk8Module(),
-          new JavaTimeModule(),
-          new KsqlParserSerializationModule(),
-          new KsqlTypesSerializationModule(),
-          new KsqlTypesDeserializationModule()
-      )
-      .enable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES)
-      .enable(DeserializationFeature.FAIL_ON_NULL_FOR_PRIMITIVES)
-      .enable(DeserializationFeature.FAIL_ON_NULL_CREATOR_PROPERTIES)
-      .enable(DeserializationFeature.FAIL_ON_INVALID_SUBTYPE)
-      .setSerializationInclusion(Include.NON_EMPTY);
+  private final ObjectMapper mapper = newInstance();
```

Comment:
we don't need this change anymore, right?

Reply:
Removed.